docs/services/endlessh.md

```diff
@@ -0,0 +1,70 @@
+# Endlessh
+
+[Endlessh-go](https://github.com/shizunge/endlessh-go) is a A golang implementation of endlessh, a ssh trapit. Installing it is powered by the [mother-of-all-self-hosting/ansible-role-endlessh](https://github.com/mother-of-all-self-hosting/ansible-role-endlessh) Ansible role.
+
+## Dependencies
+
+This service requires the following other services:
+
+- (optionally) [Traefik](traefik.md) - a reverse-proxy server for exposing endlessh publicly
+- (optionally) [Prometheus](./prometheus.md) - a database for storing metrics
+- (optionally) [Grafana](./grafana.md) - a web UI that can query the prometheus datasource (connection) and display the logs
+
+## Installing
+
+To configure and install endlessh on your own server(s), you should use a playbook like [Mother of all self-hosting](https://github.com/mother-of-all-self-hosting/mash-playbook) or write your own.
+
+## Configuration
+
+To enable this service, add the following configuration to your `vars.yml` file and re-run the [installation](../installing.md) process:
+
+```yaml
+########################################################################
+#                                                                      #
+# endlessh                                                                 #
+#                                                                      #
+########################################################################
+
+endlessh_enabled: true
+
+########################################################################
+#                                                                      #
+# /endlessh                                                                #
+#                                                                      #
+########################################################################
+```
+See the full list of options in the [default/main.yml](default/main.yml) file
+
+## Exporting Prometheus metrics
+
+Default port is 2112
```

Comment:
Maybe we need more documentation about how scraping should be configured.

None of the MASH roles automatically wire Prometheus to scrape them, and this one is no exception.

It seems like at the bare minimium, people would need to ensure Prometheus (if the one managed by the playbook is used) is on the same container network as endlessh (either by editing `endlessh_container_additional_networks_custom` or by editing `prometheus_container_additional_networks_auto`).

Although.. if both Prometheus and endlessh are exposed via Traefik, they already shared a network (`traefik`).